code/datums/gamemode/factions/legacy_cult/hell_universe.dm

```diff
@@ -103,13 +103,6 @@ In short:
 			T.underlays += "hell01"
 		CHECK_TICK
 
-	for(var/datum/lighting_corner/C in global.all_lighting_corners)
```

Comment:
is it going to be lit some other way?

Reply:
Now it is, with moody light objs. That being said, this is legacy cult, we didn't have one in years, I'm not sure if it even still works.